src/main/java/ru/skypro/homework/dto/AdsAll.java

```diff
@@ -0,0 +1,14 @@
+package ru.skypro.homework.dto;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+
+/**
+ * DTO предоставления всех объявлений
+ */
+@Data
+public class AdsAll {
+    int count;
+    ArrayList<AdDto> result;
```

Comment:
```suggestion
    List<AdDto> results;
```
названия полей в DTO должно совпадать со спецификацией, иначе работать не будет